gwswlib/tests/test_hydx.py

```diff
@@ -84,3 +148,66 @@ def test_check_on_unique(self):
             self.hydx.connection_nodes, "identificatieknooppuntofverbinding"
         )
         assert "double" in self._caplog.text
+
+
+def test_check_init_structure():
+    line_in = OrderedDict(
+        [
+            ("UNI_IDE", "ovs1"),
+            ("KWK_TYP", "OVS"),
+            ("BWS_NIV", ""),
+            ("PRO_BOK", ""),
+            ("DRL_COE", ""),
+            ("DRL_CAP", ""),
+            ("OVS_BRE", "1.5"),
+            ("OVS_NIV", "9.5"),
+            ("OVS_VOH", ""),
+            ("OVS_COE", "0.8"),
+            ("PMP_CAP", ""),
+            ("PMP_AN1", ""),
+            ("PMP_AF1", ""),
+            ("PMP_AN2", ""),
+            ("PMP_AF2", ""),
+            ("QDH_NIV", ""),
+            ("QDH_DEB", ""),
+            ("AAN_OVN", ""),
+            ("AAN_OVB", ""),
+            ("AAN_CAP", ""),
+            ("AAN_ANS", ""),
+            ("AAN_AFS", ""),
+            ("ALG_TOE", ""),
+        ]
+    )
+    line_out = {
+        "identificatieknooppuntofverbinding": "ovs1",
+        "typekunstwerk": "OVS",
+        "buitenwaterstand": None,
+        "niveaubinnenonderkantprofiel": None,
+        "contractiecoefficientdoorlaatprofiel": None,
+        "maximalecapaciteitdoorlaat": None,
+        "breedteoverstortdrempel": 1.5,
+        "niveauoverstortdrempel": 9.5,
+        "vrijeoverstorthoogte": None,
+        "afvoercoefficientoverstortdrempel": 0.8,
+        "pompcapaciteit": None,
+        "aanslagniveaubenedenstrooms": None,
+        "afslagniveaubenedenstrooms": None,
+        "aanslagniveaubovenstrooms": None,
+        "afslagniveaubovenstrooms": None,
+        "niveauverschildebiet_verhangrelatie": None,
+        "debietverschildebiet_verhangrelatie": None,
+        "aannameniveauoverstortdrempel": None,
+        "aannamebreedteoverstortdrempel": None,
+        "aannamepompcapaciteitpomp": None,
+        "aannameaanslagniveaupomp": None,
+        "aannameafslagniveaupomp": None,
+        "toelichtingregel": None,
+    }
+    structure = Structure()
+    structure.import_csvline(csvline=line_in)
+    assert structure.__dict__ == line_out
+
+
+def test_repr_structure():
```

Comment:
Ook hier weer: dit is een niet-geïnitializeerde structure. Misschien even iets erin stoppen om te kijken of de repr dat "overleeft"?